src/vic3_world/states/vic3_states_importer.cpp

```diff
@@ -15,7 +17,13 @@ vic3::StatesImporter::StatesImporter()
    database_parser_.registerRegex(commonItems::integerRegex,
        [this](const std::string& number_string, std::istream& input_stream) {
           const int state_number = std::stoi(number_string);
-          states_.emplace(state_number, state_importer_.ImportState(input_stream));
+          const auto state_string = commonItems::stringOfItem(input_stream).getString();
+          if (state_string.find("none") != std::string::npos)
```

Comment:
I prefer to search for "{" as that guarantees there's a parseable object. "none" works until some jar adds "null" and "maybe" as valid values. :D

Reply:
Good call.